server/app.js

```diff
@@ -16,11 +17,17 @@ app.use(cookieParser());
 // app.use(express.static(path.join(__dirname, "client/build")));
 
 // If you want to use a database on the cloud then just chage this URL to point to it
-const databaseUrl = "mongodb://localhost:27017/projectpaycheck";
+const databaseUrl = `mongodb://${process.env.DB_HOST}/projectpaycheck`;
```

Comment:
Can you edit the ReadMe file that tells people to make a .env file and what to include?

Reply:
English is not my native language, so feel free to edit readme or suggest what should I include